tests/dummy/app/templates/dialog/dialog-modal.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="flex items-start justify-center w-screen h-full p-12 bg-gray-50">
+  {{#let (hash isOpen=false) as |state|}}
+    <button
+      type="button"
+      class="inline-flex items-center px-4 py-2 text-base font-medium text-white bg-indigo-600 border border-transparent rounded-md shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
+      {{on "click" (set state.isOpen true)}}
+    >
+      Open dialog
+    </button>
+
+    <Dialog
+      class="fixed inset-0 z-10 overflow-y-auto"
+      @as="div"
+      @open={{state.isOpen}}
+      @onClose={{set state.isOpen false}}
+      as |d|
```

Comment:
a very minor thing, perhaps this would be more readable as `as |dialog|`?

Reply:
Can do

Reply:
Done 👍